src/cljs/rems/atoms.cljs

```diff
@@ -15,7 +15,7 @@
        :aria-label (text :t.link/download-file)}])
 
 (defn link [opts uri title]
-  [:a (merge opts {:href uri})
+  [:a (merge opts {:href uri} (select-keys opts [:on-click]))
```

Comment:
you're alreading merging opts, so adding `(select-keys opts [:on-click])` is a no-op, right?

Reply:
Indeed, I removed this commit.